docs/controller.md

```diff
@@ -0,0 +1,46 @@
+
+## Objective
+The Controller component plays a central role in the CI/CD system by orchestrating the flow of actions from the actions received, breaking these down into specific action steps, and managing communication between the Monitor, the Scheduler and the Agents.
+
+### Main features
+1. Receiving CI triggers:
+
+    a. The Controller receives CI triggers from the Monitor component.
+    b. Each trigger is analysed to determine the corresponding actions required.
+2. Breakdown into Action Steps:
+
+    a. Once an action has been received, the Controller breaks this event down into several action steps.
```

Comment:
Isn't the role of the user ?

Reply:
It needs to understand the YAML sent by the user. Maybe the 2.a. is not written clearly